src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotInverse = {};
+
+  for (const key in robot) {
+    const value = robot[key];
```

Comment:
Do we realy need these variable?

Reply:
Forgot to say that everythin else looks awesome. Nice job

Reply:
👍 
But try to give more descriptive names, not just `value`